brainfuck.py

```diff
@@ -120,7 +118,10 @@ def look_forward(insts: str, i: int) -> int:
         raise Exception("Unclosed loop.")
 
 
-class BrainfuckAssembler(vm_types.GenericAssembler):
```

Comment:
Same deal here, duck typed ... Its my bad with naming the types should probably change at some point to make Protocols clear from Super classes in the future